src/test/rustdoc-gui/shortcuts.goml

```diff
@@ -0,0 +1,25 @@
+// Check that the various shortcuts are working.
+goto: file://|DOC_PATH|/index.html
+// We first check that the search input isn't already focused.
+assert-false: "input.search-input:focus"
+press-key: "s"
+assert: "input.search-input:focus"
+press-key: "Escape"
+assert-false: "input.search-input:focus"
+// We now check for the help popup.
+press-key: "?"
+assert: ("#help", {"display": "flex"})
+press-key: "Escape"
+assert: ("#help.hidden", {"display": "none"})
```

Comment:
These aren't mutually exclusive, right? Should you also test before that no element has the `hidden` class?
```suggestion
assert: ("#help", {"display": "flex"})
assert-false: "#help.hidden"
press-key: "Escape"
assert: ("#help.hidden", {"display": "none"})
```

Reply:
Good idea!